web/package.json

```diff
@@ -47,6 +47,7 @@
     "lodash": "^4.17.21",
     "luxon": "^3.0.2",
     "match-sorter": "^6.3.1",
+    "nyc": "^17.0.0",
```

Comment:
something about yarn and bin files means this needs to be installed at top level now for merge coverage script to work.